index.html

```diff
@@ -353,14 +353,22 @@ <h1>Important Terms</h1>
 			<dd>
 			  <p>Usable by users in ways they can control. References in this document relate to <cite><a href="https://www.w3.org/TR/WCAG21/#operable"><abbr title="Web Content Accessibility Guidelines">WCAG</abbr> 2.1 Principle 2: Content must be operable</a></cite> [[WCAG21]]. See <a>Keyboard Accessible</a>.</p>
 			</dd>
-			<dt><dfn data-export="" data-dfn-for="ARIA" data-local-lt="owned|owned element's">Owned Element</dfn></dt>
+			<dt><dfn data-export="" data-lt="accessibility child|owned child|child element|child|children|child elements">Accessibility child</dfn></dt>
+			<dd>
+				<p>An 'accessibility child' is a <abbr title="Document Object Model">DOM</abbr> child of the <a>element</a>, a descendant of the <a>element</a> with only elements of role <rref>generic</rref> or <rref>none</rref> intervening, a child specified via an <pref>aria-owns</pref> relationship to the element, or a descendant of an element with role <rref>generic</rref> or <rref>none</rref> specified via <pref>aria-owns</pref> with only elements of role <rref>generic</rref> or <rref>none</rref> intervening.</p>
+			</dd>
+			<dt><dfn data-export="" data-dfn-for="ARIA" data-local-lt="owned element|owned|owned element's|owned elements">Owned Element</dfn></dt>
 			<dd>
 			  <p>An 'owned element' is any <abbr title="Document Object Model">DOM</abbr> descendant of the <a>element</a>, any element specified as a child via <pref>aria-owns</pref>, or any <abbr title="Document Object Model">DOM</abbr> descendant of the owned child.</p>
 			</dd>
 			<dt><dfn data-local-lt="owning">Owning Element</dfn></dt><!--Not used-->
 			<dd>
 			  <p>An 'owning element' is any <abbr title="Document Object Model">DOM</abbr> ancestor of the <a>element</a>, or any element with an <pref>aria-owns</pref> attribute which references the ID of the   element. </p>
```

Comment:
> * In Scenario #⁠1, the “Child node” `li` should be _hidden_ (i.e., excluded from the accessibility tree) because it has an “accessibility parent” — the intervening `div` — which is not rendered in host language terms.

> In this way, “owning elements” would become a subset of “accessibility parents”.

I think that work, but I'm not sure the distinction is needed.

## Scenario 3
```html
<ul style="display: none" aria-owns="li"></ul>
<div>
  <li id="li">Child node</li>
</div>
```
Here, the `li` is exposed because `display: none` prevents the `aria-owns` to take effect, therefore the `ul` is not an "accessibility parent" and it not being rendered doesn't matter.
Where this goes (I feel), is that we actually seem to only need "**DOM ancestor** not rendered => hidden" That is, I cannot really see a case where an "accessibility parent" would be not rendered, thus hiding the element, without that accessibility parent being a DOM ancestor. Because the "accessibility parent" is either a DOM ancestor or an `aria-own`ing element, but in the second case if it's not rendered, then it doesn't `aria-own` anything 🤔

Reply:
Two things:

1. For your PR,  @adampage, given what is said above, can you just use the term "accessibility parent" and DOM ancestor, and drop the "owning element"/"owned element" language?

2. For this PR, @smhigley, I'm realizing its a bit odd that "own child" and "accessibility child" have the same definition, but "owned parent" and "accessibility parent" are different. Is this intentional?

Reply:
re: #1 — I think so, depending on how the final language for “accessibility parent” shakes out.  One of the puzzles I need to solve in #1839 is to explain how the `<div aria-hidden="true">` in this scenario:

```html
<ul aria-owns="li"></ul>
<div aria-hidden="true">
  <li id="li">Child node</li>
</div>
```

...will _not_ cause the `<li>` to be excluded from the accessibility tree.  The current definition of “accessibility parent” will still apply to the `<div>` since it will continue to be a DOM ancestor after the `<ul>`’s `aria-owns` has taken effect.

So that’s where I figured a more tightly scoped definition of “owning element” would give me a way to distinguish: the `<div>` might still be both a DOM ancestor and an “accessibility parent” after `aria-owns` has been resolved, but it would no longer be an “owning element” and therefore its `aria-hidden="true"` would have no effect.

Reply:
My understanding was that `aria-owns` moved an element in the accessibility tree from one branch to another, so the owned `<li>` would no longer have an accessibility parent/child relationship with the `<div>`.

For the scenario with `display: none`, I'm not sure whether that works in practice. I think that would be a somewhat separate concern, related to #1839, in determining whether `aria-owns` works with e.g. `hidden` an `display: none`. My understanding is that in any case in which `aria-owns` is processed, the `<div>` is no longer the parent of the `<li>`.

@spectranaut I think "owning element" is a superset of "accessibility parent", since it already includes the entire tree, not just parent-chlid relationships. The generic clarification isn't necessary there, since it's an ancestor thing anyway and not a direct parent one. Same for "owned element", not sure why the wording is slightly different between the two.

Reply:
Got it, thank you.  I had misunderstood the scope of the accessibility parent/child relationship.  @spectranaut, I’ll do what you suggested and update #1839 to just differentiate between “accessibility parent” and “DOM ancestor”. 👍🏻